src/ks-app/src/app/containers/data/datagrid.component.ts

```diff
@@ -4,8 +4,49 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import {Component} from "@angular/core";
-import {SortOrder} from "@clr/angular";
+import {AfterViewInit, Component} from "@angular/core";
+import {
+    ClrDatagrid,
+    ClrDatagridActionBar,
+    ClrDatagridActionOverflow,
+    ClrDatagridCell,
+    ClrDatagridColumn,
+    ClrDatagridColumnToggle,
+    ClrDatagridFilter,
+    ClrDatagridFooter,
+    ClrDatagridHideableColumn,
+    ClrDatagridItems,
+    ClrDatagridPagination,
+    ClrDatagridPlaceholder,
+    ClrDatagridRow,
+    ClrDatagridRowDetail,
+    Comparator,
+    Datagrid,
+    DatagridActionBar,
+    DatagridActionOverflow,
+    DatagridCell,
+    DatagridColumn,
+    DatagridColumnToggle,
+    DatagridFilter,
+    DatagridFooter,
+    DatagridHideableColumnDirective,
+    DatagridItems,
+    DatagridPagination,
+    DatagridPlaceholder,
+    DatagridRow,
+    DatagridRowDetail,
+    Filter,
+    SortOrder,
+    State,
+    StringFilter
+} from "@clr/angular";
+
+import {ClrDatagridComparatorInterface} from "../../../../../clr-angular/data/datagrid/interfaces/comparator.interface";
```

Comment:
These interfaces should be added to the above imports and not be relative like this (IDEs do this sometimes).

Reply:
Yeah, queue up the IntelliJ jokes for these ;-)